code_challenges/code_wars/first_non_repeating_letter.py

```diff
@@ -0,0 +1,12 @@
+from collections import Counter
+
+
+def first_non_repeating_letter(word):
+    for letter in word:
+        if Counter(word)[letter] == 1:
+            return letter 
```

Comment:
What does `Counter(word)` do?  Does the case of the import tell you what kind of Python thing Counter is?

[docs: counter](https://docs.python.org/3/library/collections.html#collections.Counter)

Use counter but don't create a new one for every letter, it's an object so reuse the instance you create a single time.

Reply:
i think i fixed it. creating a variable, and then using that.